### PR TITLE
My Site Dashboard: Display small nudge on Today's Stats when there's no views 

### PR DESCRIPTION
**Pull request by ashiagr**
Parent #16256

This PR displays a small nudge on `Today's Stats` dashboard card when  a user has 0 views, 0 visitors, and 0 likes.

Dark Mode | Light Mode
-----------|----------
Screenshot 2022-04-06 at 7 22 03 PM | Screenshot 2022-04-06 at 7 22 19 PM

To test:

#### Interacting with the nudge

1. Make sure you have a site with a few views, visitors, or likes and one with zero
2. Open the app
3. Tap home
4. ✅ Check that the nudge appears for a site without views, visitors, and likes
5. Tap on the nudge
6. ✅ Make sure a webview displaying `https://wordpress.com/support/getting-more-views-and-traffic/` appears

#### Switching sites

1. Switch to a site with views, visitors and/or likes
2. ✅ Make sure the nudge is not displayed
3. Switch back to a site without any views, visitors and likes
4. ✅ Make sure the nudge is displayed

Notes: Tracking to be added as a separate sub-task.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItem.kt
Comment (zwarm) [inline WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItem.kt]: Nit pick: I know it's text, but `Text` feels very generic as the type for message. I don't have a better suggestion, just wanted to call it out so we can think about it.

**Review by zwarm — APPROVED** [review]
All works as expected. 👏  

I left one nitpick, but it won't hold up my approval, but I will leave it for you to review before merging.

**Review by osullivanchris — APPROVED** [review]
downloaded the build and checked in light and dark mode - looks great.

Didn't even know we were gonna get this done on Android so even better! 😸

File: WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItem.kt
Comment (ashiagr) [reply WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItem.kt]: You're right, actually, I kept the naming closer to Stats -> BlockListItem.Text

https://github.com/wordpress-mobile/WordPress-Android/blob/a5568fac17d88b7d669c765a531758152ec56211/WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/BlockListItem.kt#L160

But we have a different structure. I'll rename it to `TextWithLinks` till we come up with a better name, at least it is better than generic name `Text`. 😄

Comment (ashiagr) [reply WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItem.kt]: Done in [dc52482](https://github.com/wordpress-mobile/WordPress-Android/pull/16268/commits/dc52482f9f452b535681f1c160c4b3daeb403958). I'll merge the PR now.

**Comment by osullivanchris** [comment]
Sorry I know we are finished on this task but I got some feedback that the nudge could be written a bit better.

FAO editorial:
this is on the new My Site Home/Dashboard where we show dynamic content. If the user has very low stats, we'll show this nudge so that it looks a bit helpful and not just sad!

The current text is
If you want to try and get more views and traffic check out our top tips

'Top tips' is a hyperlink going to https://wordpress.com/support/getting-more-views-and-traffic/

With my attempt at the copy I said 'try' because we can't promise anything. And I ended the sentence with 'top tips' so I could have the hyperlink at the end.

**Comment by kristastevens** [comment]
Hi @osullivanchris, thanks for the ping! 

Here are a few options:  

- Would you like to grow your audience? Check out our top tips. 
- Would you like to earn more site visitors? Check out our top tips. 
- Interested in building your audience? Check out our top tips.

**Comment by osullivanchris** [comment]
@kristastevens thanks for taking a look

The third one sounds really classy to me, I'll go with that personally! Will relay to the team. Thanks again! 🙏 

> Interested in building your audience? Check out our top tips.

File: WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/todaysstats/TodaysStatsCardViewHolder.kt
Comment (AliSoftware) [inline WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/todaysstats/TodaysStatsCardViewHolder.kt:113]: This specific line:
 - Made the code crash with an out-of-bounds error in locales/translations where there were no character after the link.
 - "Ate" the character just after the link in other locales which had one, by replacing it with the drawable instead —e.g. in English, it ate the `.` after the link
 - This issue was even more impactful in locales using ideograms (e.g. Korean, Japanese, Chinese), as by "eating" the next character after the link, that replace a whole ideogram… rendering the translation nonsensical

See some more info and context in [this comment](https://github.com/wordpress-mobile/WordPress-Android/pull/16446#discussion_r862026952)

**Pull request merged**
